song/app.js

```diff
@@ -94,6 +98,54 @@ app.post("/content", async (req, res, next) => {
   );
   res.status(201).json({ message: "postCreated" });
 });
+// 좋아요 기능
+app.post("/like", async (req, res, next) => {
+  const { userId, postId } = req.body;
+  await appDataSource.query(
+    `
+    INSERT INTO likes(
+      user_id,
+      post_id
+    ) VALUES (?, ?)
+    `,
+    [userId, postId]
+  );
+
+  res.status(200).json({ message: "likeCreated" });
+});
+
+app.patch("/update", async (req, res, next) => {
+  const { content, userId } = req.body;
+  await appDataSource.query(
+    `UPDATE posts
+          SET 
+          content = ?
+          WHERE posts.user_id = ?
+          `,
+    [content, userId]
+  );
+  const update = await appDataSource.query(
+    `SELECT
+           users.id as userId,
+           users.name as userName,
+           posts.user_id as postingId,
+           posts.title as postingTitle,
+           posts.content as postingContent
+           FROM users LEFT JOIN posts ON users.id = posts.user_id     
+    `
+  );
+
+  res.status(200).json({ data: update });
+});
+
+app.delete("/delete", async (req, res, next) => {
+  const { userId } = req.body;
+  await appDataSource.query(
+    `DELETE FROM posts
+    WHERE posts.user_id = ${userId}`
+  );
+  res.status(200).json({ message: "postingDelete" });
+});
```

Comment:
```suggestion
app.delete("/posts/:postId", async (req, res) => {
  const { postId } = req.params;
  await appDataSource.query(
    `DELETE FROM posts
    WHERE posts.id = ?`, [ postId ]
  );
  
  return res.status(204).send();
});
```
지금 현재 로직으로는 해당 유저가 작성한 게시글을 전부 지우는 건데 의도하신건가요? 
클라이언트에서 요청으로 넘겨받는 하나의 게시글만 삭제 할 수 있도록 수정해주세요.

1. 위에서 언급드린 코멘트와 마찬가지로 엔드포인트에는 `delete`와 같은 동사(동작)가 들어가지 않도록 해주세요. http method 가지고 충분히 설명이 됩니다.
2. delete 메소드에는 body로 데이터를 보내지 않도록, 예시와 같이 params로 혹은 query(추후에 다뤄볼 예정입니다)로 보낼 수 있도록 수정 해주세요. 이는 실제로 delete method를 사용할 때에는 body로 들어오는 데이터를 파싱하지 않도록 설계된 프레임워크나, 버전이 있기 때문에 꼭 사용해야한다면 유의하여 사용하되, 원칙적으로는 지양하는 것이 좋습니다. 의도와 다르게 동작할 수 있습니다.
3. 가급적 불가피 한 경우가 아니라면, 템플릿 리터럴(`${}`)보다 place holder(`?`)를 사용할 수 있도록 해주세요. 이는 보안과 연관 있는데, 이후에 구두로 설명 드리겠습니다.
4. delete method에는 204라는 status code를 응답으로 더 많이 쓰고 있습니다. 다만 204는 message로 no content를 의미하고 실제로 바디에 어떠한 데이터를 담아서 보내지 않으므로 send()만 해주면 됩니다.